www/js/app.js

```diff
@@ -218,19 +220,79 @@ define(['jquery', 'zimArchiveLoader', 'util', 'uiUtil', 'cookies','abstractFiles
         return true;
     });
     // Top menu :
+    function addAnimation(section) {
```

Comment:
@vasusharma7 Sorry I wasn't clear. I meant to move this function also to uiUtil.js.

Reply:
OK, I will move that also there